source/GameInterface/Services/Clans/Patches/ClanChangeInfluencePatch.cs

```diff
@@ -16,6 +17,8 @@ public class ClanChangeInfluencePatch
 
         static bool Prefix(Clan clan, float amount)
         {
+            if (PolicyProvider.AllowOriginalCalls) return true;
+
             CallStackValidator.Validate(clan, AllowedInstance);
 
             if (AllowedInstance.IsAllowed(clan)) return true;
```

Comment:
```suggestion
            if (AllowedInstance.IsAllowed(clan)) return true;
            if (ModInformation.IsClient && clan == Clan.PlayerClain) return false;
```

Reply:
This should work well imo, maybe there's a variable in clan that can be used instead of Clan.PlayerClan